pytorch/training/docker/2.2/py3/Dockerfile.cpu

```diff
@@ -294,8 +290,8 @@ WORKDIR /root
 RUN pip install --no-cache-dir --upgrade pip --trusted-host pypi.org --trusted-host files.pythonhosted.org \
  && pip install --no-cache-dir -U \
     smclarify \
-    "sagemaker>=2,<3" \
-    "sagemaker-experiments<1" \
+    sagemaker \
```

Comment:
why are we unpinning these versions? major version changes to sagemaker sdk may come with backward incompatible changes

Reply:
For sagemaker, we want to install latest packages and then fix tests or errors if there is a major version change or pin it to a lower major version if there is a customer ask.

But I can understand wanting to pin the major versions for safety so I pin them back if you prefer

Reply:
yes let pin sagemaker versions as they are used in SM DLC images